### ISSUE TITLE
Import data from pygdsm

### ISSUE THREAD

**Issue by mkolopanis** [RadioAstronomySoftwareGroup/pyradiosky#124]
[pygdsm](https://github.com/telegraphic/pygdsm) is one of the only python wrappers for GSM data (both 2008 and 2016). Due to licensing we cannot import this module directly (depends on healpy/GPL license) but we could construct a function which coerces maps created by this module into a sky object.

**Comment by mkolopanis** [comment]
This is basically what I came up with in a jupyter notebook. Not sure how much more we would need? Here Galactic is an astropy coordinates object that you can just import from the module.

```
gsm2008_coords = HEALPix(ahp.npix_to_nside(gsm2008_cube.shape[1]), frame=Galactic())
coords = gsm2008_coords.healpix_to_skycoord(np.arange(gsm2008_coords.npix))
coords_icrs = coords.transform_to("icrs")
ra, dec = coords_icrs.ra, coords_icrs.dec
skymod_freqs  = freqs * units.Hz
stokes = units.Quantity(np.zeros((4, freqs.size, gsm2008_coords.npix)), unit=units.K)
stokes[0] = gsm2008_cube << units.K
```

**Comment by aelanman** [comment]
I think this is a specific case of issue #119 . We've discussed adding frame transformations to our setup.

**Comment by mkolopanis** [comment]
yeah this very well could be. It just came up again today and felt at least different enough to me at the time :thinking:

**Comment by bhazelton** [comment]
@mkolopanis now that we've implemented the frame support, is this still something we should do?
